test/e2e/wireguard_test.go

```diff
@@ -37,24 +37,24 @@ func TestWireGuard(t *testing.T) {
 
 	data, err := setupTest(t)
 	skipIfEncapModeIsNot(t, data, config.TrafficEncapModeEncap)
```

Comment:
To ensure the namespace is deleted, this line should be moved after defer too.

Reply:
updated